pkg/crc/network/utils.go

```diff
@@ -13,13 +13,13 @@ import (
 	crcos "github.com/code-ready/crc/pkg/os"
 )
 
-func executeCommandOrExit(sshRunner *ssh.SSHRunner, command string, errorMessage string) string {
+func executeCommandOrExit(sshRunner *ssh.SSHRunner, command string, errorMessage string) (string, error) {
```

Comment:
This helper can probably be removed, and sshRunner.Run be called directly? At least it should be renamed.

Reply:
yes done